### ISSUE TITLE
FR: TV Shows "You May Like" Intelligence

### ISSUE THREAD

**Issue by DrSeussFreak** [petio-team/petio#334]
**Is your feature request related to a problem? Please describe.**
When browsing the TV section of Petio, I really enjoy the 2 sections for "shows you may like", but it would be great if the shows I already have (which Petio clearly marks with the green corner) were filtered out, since, well, I already have it.

**Describe the solution you'd like**
Filter out existing shows from the "shows you may like" area

**Describe alternatives you've considered**
Add a toggle switch to give the user the choice to do this

**Additional context**
The other thing I see is that for both of the sections visible on my page (animation shows, action shows) they are 90% all anime (see below)

![Screen Shot 2021-04-02 at 9 47 53 AM](https://user-images.githubusercontent.com/175500/113426046-8031ee00-9398-11eb-94aa-99029385f5ce.png)

**Comment by Dulanic** [comment]
Agree, only saying agree as I mentioned it for movies. Just need to expand scope.
